camera/imagecapture.c

```diff
@@ -275,19 +286,20 @@ int main(int argc, char *argv[]){
         camctrl(fd, VIDIOC_DQBUF, &vidbuffer);
 
         //Prepare jpeg outputs
-        sprintf(jpegname, "image%03d.jpeg", i);
+        sprintf(jpegname, "image%03d.ppm", i);
         fout = fopen(jpegname, "w");
         if(!fout) {
             printf("OUTPUT ERROR: Cannot create jpeg\n");
             exit(EXIT_FAILURE);
         }
         //Fill jpeg files with frame data
-        //Placeholder Jpeg header data
         fprintf(fout, "P6\n%d %d 255\n",format.fmt.pix.width, format.fmt.pix.height);
         fwrite(buffers[vidbuffer.index].start, vidbuffer.bytesused, 1, fout);
         fclose(fout);
```

Comment:
If we're writing PPMs instead of JPGs, let's change variable names and comments to match.